images/build/cross/variants.yaml

```diff
@@ -1,25 +1,35 @@
 variants:
+  dev:
+    TYPE: 'default'
+    CONFIG: 'dev'
+    GO_VERSION: '1.16.5'
+    IMAGE_VERSION: 'v1.22-dev-1'
+    PROTOBUF_VERSION: '3.7.0'
+    ETCD_VERSION: 'v3.5.0'
```

Comment:
remind me why etcd is included in kube-cross?

Reply:
possibly the the integration tests dockerized (e.g. on mac)?
would like to see this verified or we can just remove it.

Reply:
@BenTheElder -- https://github.com/kubernetes/release/pull/2124 builds an image without etcd, but I haven't had time this week to test just yet.

Reply:
Testing here: https://github.com/kubernetes/kubernetes/pull/104400